### DISCUSSION TITLE
Full e2e social sign-in example

### DISCUSSION THREAD

**Discussion by ghost** [ory/kratos#2758] [Q&A] [answered]
Is there a good end-to-end example for OIDC flow with social providers when user say hits "Sign in with Google"?

The provided documentation does not go beyond configuring social providers and just describes on the high level how it works without concrete full examples ... details matter

Tried searching and there is none that can be found

Thank you

**Comment by vinckr** [comment, marked as answer]
There is a [jsonnet snippet](https://github.com/ory/kratos/tree/master/contrib/quickstart/kratos/oidc) that works with the [quickstart](https://www.ory.sh/docs/kratos/quickstart). Example configuration is [here](https://www.ory.sh/docs/kratos/social-signin/github). 
Would it help to tie all of those together in a concrete example? https://github.com/ory/examples would probably be a good place for that! 

Can you share a bit more what details you were missing?

**Comment by ghost** [comment]
Thank you for your reply. I will add angular example with a full end to end demo with explanations. In the end I worked it out myself by trial and error where things weren't 100% clear. Happy to send that example to you should you be interested as it will save a lot of time for people like me :-)

**Reply by vinckr** [reply]
That would be awesome if you contribute it to https://github.com/ory/examples !
It would help newcomers a lot (and I would also love to learn more angular 😉 )
We just completed a batch of new swag and I would be happy to send you something as a small thank you!

**Reply by manikanta47052** [reply]
HI @8thblock can you give an example how to integrate angular with kratos if you already implemented. I implemented google in my angular as well as kratos container but not able to integrate both. Thanks in advance.

**Reply by manikanta47052** [reply]
@vinckr do u know where is this full e2e social sign in example available, provided by @8thblock ?

**Reply by vinckr** [reply]
Not yet. Take a look at this PR: https://github.com/ory/docs/pull/1210 (will be merged soon), maybe it helps until we have a proper Angular example.

**Reply by manikanta47052** [reply]
@vinckr just one doubt actually im not using ORY console or CLI, Is kratos supports integrating google social sign in with out using Console or Cli.

**Reply by vinckr** [reply]
Hello @manikanta47052 
Yes if you are self-hosting Ory Kratos, you can use the CLI that comes with it. To add social sign in you need to edit the configuration directly.
